tests/model_builder/test_model_builder.py

```diff
@@ -183,6 +214,7 @@ def test_fit(fitted_model_instance):
     post_pred[fitted_model_instance.output_var].shape[0] == prediction_data.input.shape
 
 
+@pytest.mark.slow
```

Comment:
Just reduce tune and draws, specially since nothing is being checked other than the trace exists after the fact